battles/urls.py

```diff
@@ -1,13 +1,15 @@
 from django.conf.urls import url
 
-from .views import BattlesListView, BattleView, ChoosePokemonTeamView, CreateBattleView
+from .views import (
+    BattlesListView, BattleView, ChoosePokemonTeamView, CreateBattleView, PokemonAutocompleteView
+)
 
 
 urlpatterns = [
     url(r'^$', BattlesListView.as_view(), name='list'),
 
     url(r'^create$', CreateBattleView.as_view(), name='create-battle'),
     url(r'^details/(?P<pk>[\w-]+)$', BattleView.as_view(), name='details'),
-    url(r'^details/(?P<pk>[\w-]+)/team$',
-        ChoosePokemonTeamView.as_view(), name='team')
+    url(r'^details/(?P<pk>[\w-]+)/team$', ChoosePokemonTeamView.as_view(), name='team'),
+    url(r'^pokemon-autocomplete$', PokemonAutocompleteView.as_view(), name='pokemon-autocomplete')
```

Comment:
maybe a better url for this endpoint would be '^api/pokemon/$', and the name api-pokemon-list. This library implements a RESTful list api, so it would be better if it looked more like that